tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -1664,6 +1664,47 @@ public function data_next_tag_ignores_script_tag_contents() {
 		);
 	}
 
+	/**
+	 * Invalid tag names are comments on tag closers.
+	 *
+	 * See https://html.spec.whatwg.org/#parse-error-invalid-first-character-of-tag-name
+	 *
+	 * @ticket 58007
+	 *
+	 * @dataProvider data_next_tag_ignores_invalid_first_character_of_tag_name_comments
```

Comment:
```suggestion
	 *
	 * @covers WP_HTML_Tag_Processor::next_tag
	 *
	 * @dataProvider data_next_tag_ignores_invalid_first_character_of_tag_name_comments
```
See [this ticket](https://core.trac.wordpress.org/ticket/56070) as a reference for the test docblock order/spacing of annotations.

Reply:
spaced out in 8813327002